File: task.py
n = int(input())
numbers = list(map(int, input().split()))

for x in numbers:
    if x < 10:
        continue
    
    last_digit = x % 10
    temp = x
    good = True
    
    while temp > 0:
        if temp % 10 != last_digit:
            good = False
            break
        temp = temp // 10
    
    if good:
        print(x, end=" ")
print()

File: task.js
const fs = require('fs');
const input = fs.readFileSync(0, 'utf8').trim().split(/\s+/);

let index = 0;
const n = parseInt(input[index++]);

for (let i = 0; i < n; i++) {
    const x = parseInt(input[index++]);
    
    if (x < 10) {
        continue;
    }
    
    const lastDigit = x % 10;
    let temp = x;
    let good = true;
    
    while (temp > 0) {
        if (temp % 10 !== lastDigit) {
            good = false;
            break;
        }
        temp = Math.floor(temp / 10);
    }
    
    if (good) {
        process.stdout.write(x + " ");
    }
}
console.log();


